scripts/lib/libvirt/compute-config

```diff
@@ -32,6 +32,8 @@ def main():
                         help="Virtual Disk Directory (e.g /dev/cloud)")
     parser.add_argument("bootorder", type=str,
                         help="Boot Order (e.g. 2)")
+    parser.add_argument("numcontrollers", type=str,
+                        help="Number of controller nodes")
```

Comment:
The tests for this code expects all the arguments to be [strings](https://github.com/SUSE-Cloud/automation/blob/master/scripts/lib/libvirt/test_libvirt_setup.py#L18). I would tackle that str->int in another PR.